webapp/channels/src/components/markdown_image/index.ts

```diff
@@ -5,9 +5,23 @@ import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 import type {Dispatch} from 'redux';
 
+import {getPost} from 'mattermost-redux/selectors/entities/posts';
+
 import {openModal} from 'actions/views/modals';
 
+import type {GlobalState} from 'types/store';
+
 import MarkdownImage from './markdown_image';
+import type {Props} from './markdown_image';
+
+function mapStateToProps(state: GlobalState, ownProps: Props) {
+    const post = getPost(state, ownProps.postId);
+    const isUnsafeLinksPost = post?.props?.unsafe_links !== '';
```

Comment:
Can we make this a check for `=== 'true'` instead of a check for `!== ''`, my concern again is about `unsafe_links === 'false'` being true. Can easily end up being a bug in a plugin side where you set the value to "true" or to "false" depending on certain criteria.